2.4/test/run

```diff
@@ -25,10 +24,8 @@ function run() {
     cmd="$1"
     expected_res="${2:-0}"
     msg="${3:-Running command '$cmd'}"
-    set +e
     run_command "$cmd" "$expected_res" "$msg"
```

Comment:
It would be nice to get rid of `run()` function and `run_command` it is too complicated from my point of view and too general. But this task is for another pull request.

Reply:
I agree. I can look into it when I finish with this.